docs/reference/object_tree.md

```diff
@@ -0,0 +1 @@
+::: squape.object_tree
```

Comment:
What about filters for some of the pages to remove private methods?

options:
        filters: ["!^_"]